src/components/not-found.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { NimbleEmoji } from '.'
+
+export default class NotFound extends React.PureComponent {
+  constructor(props) {
+		super(props)
+		this.state = { errored: false }
+
+		this.handleError = this.handleError.bind(this)
+	}
+
+	defaultImage() {
+		var { data, emojiProps } = this.props
+
+		return (
+			<div>
+				{NimbleEmoji({
+					data: data,
+					...emojiProps,
+					size: 38,
+					emoji: 'sleuth_or_spy',
+					onOver: null,
+					onLeave: null,
+					onClick: null,
+				})}
+			</div>
+		)
+	}
+
+	handleError(event) {
+		this.setState({ errored: true })
+	}
+
+  render() {
+		var { i18n, notFoundImg } = this.props
+
+		if (this.state.errored) {
+			return (
+				<div>
+					{ this.defaultImage() }
+					<div className="emoji-mart-no-results-label">{i18n.notfound}</div>
+				</div>
+			)
+		}
+
+		return (
+			<div>
+				<img src={notFoundImg} onError={this.handleError}/>
+				<div className="emoji-mart-no-results-label">{i18n.notfound}</div>
+			</div>
+		)
+	}
+}
```

Comment:
We should add `propTypes` for this component. See the bottom of all the other components for examples. 
This helps document the component, as well as providing some validation that the inputs we're providing are what we expect.